scipy/stats/_continuous_distns.py

```diff
@@ -6727,6 +6997,9 @@ class rayleigh_gen(rv_continuous):
     """
     _support_mask = rv_continuous._open_support_mask
 
+    def _shape_info(self):
+        return []
+
```

Comment:
OK, I checked from the bottom up to here.
One thought is to add a check that tests that `_shape_info` is defined (and consistent with `_argcheck`?) for all distributions.